commands/command_fetch.go

```diff
@@ -107,11 +107,27 @@ func fetchCommand(cmd *cobra.Command, args []string) {
 }
 
 func pointersToFetchForRef(gitscanner *lfs.GitScanner, ref string) ([]*lfs.WrappedPointer, error) {
-	pointerCh, err := gitscanner.ScanTree(ref)
-	if err != nil {
+	var pointers []*lfs.WrappedPointer
+	var multiErr error
+	tempgitscanner := lfs.NewGitScanner(func(p *lfs.WrappedPointer, err error) {
+		if err != nil {
+			if multiErr != nil {
+				multiErr = fmt.Errorf("%v\n%v", multiErr, err)
```

Comment:
I added `errors.Combine` in my original locking PR so 👍 (although it only concatenated with a line break as you're doing here, nothing smart like storing inner errors)